app/assets/stylesheets/description_box.scss

```diff
@@ -1,8 +1,8 @@
-.description {
-    border: 1px #333;
+.attribute {
+    border: 1px solid;
     padding:5px;
 }
-.planet-text {
+.description {
     display:none;
-    border:1rem;
+    text-shadow: 2px 2px #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces